src/main/java/org/example/application/CountIncomeTaxService.java

```diff
@@ -0,0 +1,87 @@
+package org.example.application;
+
+import lombok.AllArgsConstructor;
+import org.example.domain.customer.Customer;
+import org.example.domain.customer.TaxPayments.FlatTax;
+import org.example.domain.customer.TaxPayments.GeneralTax;
+import org.example.domain.customer.TaxPayments.LumpSumTax;
+import org.example.domain.customer.TaxPayments.TaxPaymentForm;
+import org.example.domain.money.Currency;
+import org.example.domain.money.Money;
+import org.example.port.in.invoice.CountIncomeTaxUseCase;
+
+import java.math.BigDecimal;
+import java.time.YearMonth;
+
+import static org.example.application.CountHealthInsuranceContributionService.FIRST_CONTRIBUTION_RANGE;
+import static org.example.application.CountHealthInsuranceContributionService.SECOND_CONTRIBUTION_RANGE;
+
+@AllArgsConstructor
+
+public class CountIncomeTaxService implements CountIncomeTaxUseCase {
+    private CountMonthlyMonthlyRevenueService countMonthlyRevenueService;
```

Comment:
znow nie wiem czy nie powinien tu byc uzyty customer a jak cos to tutaj powinien isc interfejs nie implementacja